casbin/util/split.cpp

```diff
@@ -34,8 +34,6 @@ std::vector<std::string> Split(std::string str, const std::string& del, int limi
     if(limit <= 0)
         limit = LARGE;
 
```

Comment:
```suggestion
    tokens.reserve((limit == LARGE) ? 0 : limit);
```

I intended to reserve space for strings beforehand depending on the upper-bound of the current limit.

In case the `limit == LARGE` is true, we must ditch reserving space altogether as it should be completely dynamic.